lib/formulator.ex

```diff
@@ -83,16 +93,37 @@ defmodule Formulator do
   defp build_input(form, field, options, error) do
     input_type = options[:as] || :text
     input_class = options[:class] || ""
+
     options =
       options
+      |> add_validation_attributes(form, field)
+      |> add_format_validation_attribute(form, field)
       |> Keyword.delete(:as)
+      |> Keyword.delete(:validate)
+      |> Keyword.delete(:validate_regex)
       |> Keyword.put(:class, add_error_class(input_class, error.class))
 
     apply(Phoenix.HTML.Form, input_function(input_type), [form, field, options])
   end
 
+  defp add_validation_attributes([validate: true] = options, form, field) do
+    Phoenix.HTML.Form.input_validations(form, field)
```

Comment:
What do you think about starting the pipe here with a bare term?

`form |> Phoenix.html |> `